transmitter/src/transmitter.cpp

```diff
@@ -0,0 +1,59 @@
+#include "transmitter.h"
+#include "controls.h"
+
+
+// REPLACE WITH THE MAC Address of your receiver
+uint8_t recmac[]             = { 0x08, 0xB6, 0x1F, 0x29, 0x9A, 0x8C };
+uint8_t transmac[]           = { 0x3C, 0x71, 0xBF, 0x9D, 0x9F, 0x88 };
+unsigned long previousMillis = 0;
```

Comment:
```suggestion
unsigned long previous_millis = 0;
```

Reply:
are you really reviewing my development code

how bored were you :D